assets/js/oath-parser.js

```diff
@@ -4,8 +4,9 @@ import * as other from './other.js'
 
 let worldDeckSize, dispossessedDeckSize, relicDeckSize;
 const format = []
-const worldSuitArray = [0,0,0,0,0,0]
-const dispSuitArray = [0,0,0,0,0,0]
+const suits = ['discord', 'hearth', 'nomad', 'arcane', 'order', 'beast']
+const worldSuitDict = {'discord': 0, 'hearth': 0, 'nomad': 0, 'arcane': 0, 'order': 0, 'beast': 0}

```

Comment:
@AndBM You should take a look at the changes here for the future in case you are updating the JS again later